elastic/kibana
# [Security Solution] Removes outdated rule tour for rule management workflow

## Summary

Removes the tour for rule features implemented in https://github.com/elastic/kibana/pull/176767 as they are no longer new. Keeps the component and logic to use for future tours, just removes reference to the specific rules management tour.

### Steps to test

1. Make sure the `securitySolution.rulesManagementPage.newFeaturesTour.v8.13` local storage key is cleared from your kibana page (this will make the tour show up normally)
2. Notice that navigating to the rule management page does not display the rule tour
